galley/pkg/runtime/processing/dispatcher.go

```diff
@@ -23,13 +23,21 @@ var scope = log.RegisterScope("processing", "Galley data processing", 0)
 
 // Dispatcher is a Handler that can dispatch events to other Handlers, based on Collection.
 type Dispatcher struct {
-	handlers map[resource.Collection][]Handler
+	fullSyncHandlers []Handler
+	handlers         map[resource.Collection][]Handler
 }
 
 var _ Handler = &Dispatcher{}
 
 // Handle implements Handler
 func (d *Dispatcher) Handle(e resource.Event) {
+	if e.Kind == resource.FullSync {
```

Comment:
This should be handled at the processor level.

Reply:
fixed.